packages/theme-chalk/src/switch.scss

```diff
@@ -123,6 +124,13 @@ $switch-button-size: map.merge(
     height: 0;
     opacity: 0;
     margin: 0;
+
+    &:focus-visible {
+      & + .#{$namespace}-switch__core {
```

Comment:
Looks like the style isn't working when the switch has `active-text`, this should fix:

`& ~ .#{$namespace}-switch__core {`

Reply:
Fixed!